### PR TITLE
feat: Add eslint

### PR DESCRIPTION
**Pull request by saranshdhingra**

### CODE REVIEW

**Comment by crwilcox** [comment]
This PR title seems very misleading as what I see is an authoring of many react components?

**Comment by enocom** [comment]
We could probably land this PR as just adding eslint. Also, I'd love to see prettier in here as well.

**Comment by saranshdhingra** [comment]
> This PR title seems very misleading as what I see is an authoring of many react components?

Updated the base branch, this should make the review process better. But we'll have to change the merge order now :)

**Comment by saranshdhingra** [comment]
> We could probably land this PR as just adding eslint. Also, I'd love to see prettier in here as well.

Sure! I would send it after this is merged. Best to keep the PR chains to a minimum I would say :)

File: web/src/js/components/about/AboutContainer.js [hunk 1]
```diff
@@ -14,42 +14,71 @@
  * limitations under the License.
  */
 
-import React from "react";
-
+import React from 'react';
```
Comment (crwilcox) [inline web/src/js/components/about/AboutContainer.js:17]: It seems elements of this PR are still related to a react change and not adding eslint?

Comment (saranshdhingra) [reply web/src/js/components/about/AboutContainer.js:17]: I think this change is shown due to the eslint rule for using consistent quotes. So the change here is the modification from double quotes to single quotes.

**Comment by enocom** [comment]
I think what's confusing here is that this PR includes the React change.

Could we start with just adding eslint to the existing JavaScript as a PR and then discuss the React change separately?

**Comment by crwilcox** [comment]
Since I think this is going to be hard to unravel, I just started over with eslint and the prettier plugin: https://github.com/GoogleCloudPlatform/gcping/pull/75

**Pull request closed without merging**